src/libsystemd-network/sd-dhcp-client-id.c

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "dhcp-client-id-internal.h"
+#include "unaligned.h"
+#include "utf8.h"
+
+int sd_dhcp_client_id_clear(sd_dhcp_client_id *client_id) {
```

Comment:
we usually call this _done() or _reset(), no? (didn't check, we might be a bit chaotic there)

Reply:
`_done()` is usually (not always) used when the object will not be reused.
`_clear()` makes the passed object reused.
But, such rule is not uniformly applied in existing code, IIRC.
Personally, I prefer `_clear()` here.